### PR TITLE
fix: 🐛 update whats new cards

### PR DESCRIPTION
**Pull request by paulstey**
This PR just refreshes the `What's New` section on the main page. I also added a caveat bullet on our `Rates` page.

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [x] :frog: Data (data folder - people/opportunities)
- [ ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [ ] Commitizen was used for all commits.
- [ ] Local site looks as expected after changes.
- [ ] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

File: content/english/services/rates.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ lead: We provide services with limited resources at **no cost** to all
 
 {{< rates >}}
 
+ * Note, these values are subject to period review and changes
```
Comment (mcmcgrath13) [inline content/english/services/rates.md]: `period` => `periodic`

File: data/whatsnew.yaml [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+  - title: We're Hiring!!!
+    text: We have several open positions at the Center for Computation and Visualization! [**Check out opportunities.**](about/#opportunities)
     fa:
       prefix: fad
-      icon: fa-database
-  - title: COVID-19 Updates
-    text: The Center for Computation and Visualization remains committed to supporting the computing needs of the Brown Research Community during this difficult time. [**Click here for details.**](covid/)
+      icon: fa-door-open
+  - title: Bootcamp
+    text: CCV will be hosing a virtual bootcamp for HPC and computational biology! This will be the first week in May; so check back for details!
```
Comment (mcmcgrath13) [inline data/whatsnew.yaml]: would be really good to have something to link to here - registration? the ccv events page?

**Review by mcmcgrath13 — CHANGES_REQUESTED** [review]

File: data/whatsnew.yaml (hunk 2 above)
Comment (paulstey) [reply data/whatsnew.yaml]: I'll add this in a subsequent PR. We just need to get the word out.

**Review by singhsaluja — APPROVED** [review]

File: content/english/services/rates.md (hunk 1 above)
Comment (paulstey) [reply content/english/services/rates.md]: Fixed! Thank you, @mcmcgrath13!!

**Review by mcmcgrath13 — APPROVED** [review]

**Comment by paulstey** [comment]
Thanks, @mcmcgrath13 and @singhsaluja!!!

**Pull request merged**